lib/pages/chat_list/chat_list.dart

```diff
@@ -159,10 +161,26 @@ class ChatListController extends State<ChatList>
     }
   }
 
+  String getHostName() {
+    final client = Matrix.of(context).client;
+    final userId = client.userID;
+    final userHostName = extractHostName(userId!.split(':')[1]);
+    return userHostName;
+  }
+
+  List<String> getBotIds() {
+    final hostName = getHostName();
+    return SocialNetworkManager.socialNetworks.map((sn) => sn.chatBot + hostName).toList();
+  }
+
+  // List of user or bot IDs to exclude
+  List<String> get excludedUserIds => getBotIds();
```

Comment:
Performance : is there a way to calculate this only once, instead of at every call ?